src/ansiblelint/testing/fixtures.py

```diff
@@ -50,6 +52,15 @@ def rule_runner(request):
     return RunFromText(collection)
 
 
+@pytest.fixture
+def config_options():
+    """Return configuration options that will be restored after testrun."""
+    global options  # pylint: disable=global-statement
```

Comment:
I am not happy with this fixture, probably the better idea will be to create some `ConfigWrapper` like for instance `pytest-django` do in the `settings` fixture ([reference](https://github.com/pytest-dev/pytest-django/blob/59fdb49fbc380bc09bcac65de8acab767194b553/pytest_django/fixtures.py#L352)).
Do you have any other ideas for a more elegant solution?

Reply:
I think that django approach is considerably better than what we have but if you want to do this, please do it as a separated pull-request after we sort the rest.